include/RAJA/util/PluginStrategy.hpp

```diff
@@ -9,26 +9,30 @@
 #define RAJA_PluginStrategy_HPP
 
 #include "RAJA/util/PluginContext.hpp"
+#include "RAJA/util/PluginOptions.hpp"
 #include "RAJA/util/Registry.hpp"
 
 namespace RAJA {
 namespace util {
 
-
 class PluginStrategy
 {
   public:
     PluginStrategy();
 
     virtual ~PluginStrategy() = default;
 
+    virtual void init(PluginOptions p);
+
     virtual void preCapture(PluginContext p);
 
     virtual void postCapture(PluginContext p);
```

Comment:
Should these be pass by reference?

Reply:
They definitely can be. It might cause some weird issues if some RAJA plugin messes with the kID unsigned long in the PluginContext, but I'm pretty sure each plugin gets its own copy anyways so it should be alright.